A journey towards SpeakerTravel - Building a service from scratch (#342)
maartenba/maartenba.github.io: _posts/2021-11-08-todo.markdown

---
layout: post
title: "TODO come up with a title"
date: 2021-11-08 03:44:05 +0100
comments: true
published: true
categories: ["post"]
tags: ["General", ".NET", "Travel"", "Startup"]
author: Maarten Balliauw
---

> For close to two years now, I've had [SpeakerTravel](https://speaker.travel/) online. It's a tool that helps conference organizers to book flights for speakers. You invite speakers, they pick their flight of choice (within a budget the organizer can specify), and the organizer can then approve and book the flight with a single click.

Reviewer: This might read a bit better

```suggestion
For close to two years now, I've had [SpeakerTravel](https://speaker.travel/) up & running. It's a tool that helps conference organizers to book flights for speakers. You invite speakers, they pick their flight of choice (within a budget the organizer can specify), and the organizer can then approve and book the flight with a single click.
```

> In this post, I want to go a bit into the process of building this tool. Background, the technical side, the business side, and maybe a follow-up post that covers any questions you may have after reading.

Reviewer: Just a suggestion 
```suggestion
In this post, I want to go a bit into the process of building this tool. Why I started it, how it works, look at it from the business side, and maybe a follow-up post that covers any questions you may have after reading.
```

{% include toc %}

There's also a table of contents, so brace yourself for a longread!

> ## Background

Reviewer: Suggestion 
```suggestion
## Why I started building Speaker Travel
```

> Before COVID threw a wrench in offline activities, our [user group](https://www.azug.be/) was organizing [CloudBrew](https://www.cloudbrew.be/), a 2-day conference with speakers from across Europe.

Reviewer: Sometimes had speakers from US as well, but not so relevant I guess

Author: Rewriting a bit, thanks!
